sdk/template/Azure.Template/src/README.md

```diff
@@ -0,0 +1,104 @@
+# README.md template
+
```

Comment:
Given this is new we should refresh the content with the latest template at https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-EXAMPLE.md.

Reply:
Actually it isn't new it is just being added to the src directory. Is there a reason for that?

Reply:
Nope, my bad